cleaning_services/__manifest__.py

```diff
@@ -0,0 +1,58 @@
+{
+    'name': 'Cleaning Services',
+    'version': '1.0',
+    'category': 'Services',
+    'description': '',
+    'author': 'Odoo S.A.',
+    'depends': [
+        'account_online_synchronization',
+        'crm_sale_subscription',
+        'hr',
+        'industry_fsm_sale_report',
```

Comment:
This should be enough. I suspect that it was auto-installed with industry_fsm and web_studio.
```suggestion
        'industry_fsm',
```

Reply:
It is needed as the fields (`allow_worksheets`,`allow_material`,`allow_billable`) is used in `project.project` file

Reply:
Then keep `industry_fsm_sale`, and remove `allow_worksheets` as we don't use it